cypress/integration/core-opensearch-dashboards/opensearch-dashboards/apps/query_enhancements/saved_queries_old_ui.spec.js

```diff
@@ -0,0 +1,105 @@
+  INDEX_WITH_TIME_1,
+  INDEX_WITH_TIME_2,
+  SECONDARY_ENGINE,
+} from '../../../../../utils/constants';
+
+import {
+  workspaceName,
+  datasourceName,
+  setSearchConfigurations,
+  setDatePickerDatesAndSearchIfRelevant,
+  verifyDiscoverPageState,
+} from './utils/saved_search';
+
+import { generateAllTestConfigurations } from './utils/saved_queries';
+
+// This spec assumes data.savedQueriesNewUI.enabled is false.
+
+export const runSavedQueriesOldUITests = () => {
+  describe('saved queries old UI', () => {
+    before(() => {
+      // Load test data
+      cy.setupTestData(
+        SECONDARY_ENGINE.url,
+        [
+          `cypress/fixtures/query_enhancements/data-logs-1/${INDEX_WITH_TIME_1}.mapping.json`,
+          `cypress/fixtures/query_enhancements/data-logs-2/${INDEX_WITH_TIME_2}.mapping.json`,
+        ],
+        [
+          `cypress/fixtures/query_enhancements/data-logs-1/${INDEX_WITH_TIME_1}.data.ndjson`,
+          `cypress/fixtures/query_enhancements/data-logs-2/${INDEX_WITH_TIME_2}.data.ndjson`,
+        ]
+      );
+      // Add data source
+      cy.addDataSource({
+        name: datasourceName,
+        url: SECONDARY_ENGINE.url,
+        authType: 'no_auth',
+      });
+
+      // Create workspace
+      cy.deleteWorkspaceByName(workspaceName);
+      cy.visit('/app/home');
+      cy.createInitialWorkspaceWithDataSource(datasourceName, workspaceName);
+      cy.createWorkspaceIndexPatterns({
+        workspaceName: workspaceName,
+        indexPattern: INDEX_PATTERN_WITH_TIME.replace('*', ''),
+        timefieldName: 'timestamp',
+        dataSource: datasourceName,
+        isEnhancement: true,
+      });
+    });
+
+    after(() => {
+      // No need to explicitly delete all saved queries as deleting the workspace will delete associated saved queries
+      cy.deleteWorkspaceByName(workspaceName);
+      // // TODO: Modify deleteIndex to handle an array of index and remove hard code
+      cy.deleteDataSourceByName(datasourceName);
+      cy.deleteIndex(INDEX_WITH_TIME_1);
+      cy.deleteIndex(INDEX_WITH_TIME_2);
+    });
+
+    const testConfigurations = generateAllTestConfigurations();
+    testConfigurations.forEach((config) => {
+      it(`should successfully create a saved query for ${config.testName}`, () => {
+        cy.navigateToWorkSpaceSpecificPage({
+          workspaceName,
+          page: 'discover',
+          isEnhancement: true,
+        });
+
+        cy.setDataset(config.dataset, datasourceName, config.datasetType);
+
+        cy.setQueryLanguage(config.language);
+        setDatePickerDatesAndSearchIfRelevant(config.language);
+
+        setSearchConfigurations(config);
+        verifyDiscoverPageState(config);
+        cy.saveQueryOldUI(config.saveName);
+      });
+    });
+    it('should see all saved queries', () => {
+      cy.getElementByTestId('saved-query-management-popover-button').click({
+        force: true,
+      });
+
+      //Wait for queries to load
+      cy.wait(2000);
```

Comment:
same comment as above (instead of waiting arbitrary, can we wait for some element to load)

Reply:
This was left over from when I copied your structure for saved search. It doesn't seem needed for saved queries. Removing.